layouts/partials/home/graduation.html

```diff
@@ -0,0 +1,38 @@
+<section class="section has-background-white-bis graduation">
+  <div class="container">
+    <div class="columns is-vcentered">
+      <div class="column">
+        <p class="title is-size-3 is-size-4-mobile has-text-weight-light">
+          containerd has graduated from the <a href="https://cncf.io">CNCF</a>!
```

Comment:
/s/has graduated from CNCF/is now a CNCF graduated project/

IOW we are not "leaving".. :-)